core/src/mindustry/ui/dialogs/MapsDialog.java

```diff
@@ -213,4 +281,21 @@ void showMapInfo(Map map){
 
         dialog.show();
     }
+
+    @Override
+    public Dialog show(){
+        super.show();
+
+        if(Core.app.isDesktop()){
+            focusSearchField();
+        }
+
+        return this;
+    }
+
+    public void focusSearchField(){
```

Comment:
This should be inlined - the implementation in SchematicDialog was just dead code.

Reply:
Done